Makefile

```diff
@@ -1453,45 +1452,21 @@ datasets/fiwiki.labeled_revisions.20k_2016.json: \
 		datasets/fiwiki.autolabeled_revisions.20k_2016.json
 	./utility merge_labels $^ | shuf > $@
 
-datasets/fiwiki.flaggedrevs_training.65k.json: \
+datasets/fiwiki.combined_labeled_revisions.w_cache.230k_2016.json.bz2: \
```

Comment:
It's been a while, but wasn't there a reason we were using join_observations?  Are we sure that labeled_revisions and flaggedrevs_autolabeled_unreverted are disjoint?  Is it okay to have some revisions repeated?

Reply:
Hmm... Probably OK unless it is super likely there's a lot of overlap.  I'll check.